src/components/pages/Login/_LoginContainerStyled.less

```diff
@@ -98,9 +98,24 @@
   }
 
   @media (max-width: 535px) {
+    width: 60%;
```

Comment:
Initial width was 80% but changed to 60. Github interpreted oddly because below I added a new media query that did use 80%.